samcli/local/apigw/local_apigw_service.py

```diff
@@ -701,6 +702,8 @@ def _invoke_parse_lambda_authorizer(
         route: Route
             The route that is being called
         """
+        EventTracker.track_event(EventName.USED_FEATURE.value, UsedFeature.CUSTOM_LAMBDA_AUTHORIZERS.value)
```

Comment:
why we do not track if the lambda authorizer works successfully or not ?

Reply:
I decided to track all uses of authorizers, whether or not they were successful or not since we can filter the results out. If we typically track after success, then I can change the call to be after, no issues with either approach